README.md

```diff
@@ -1,16 +1,162 @@
-# Title
+# Return Values
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+- Identify implicit return values in Ruby syntax
+- Recognize the explicit `return` keyword
 
 ## Introduction
 
-## SWBAT 1
+Displaying text in the console is not the only way available to get information
+from our Ruby methods. While `puts` and `print` display information, they don't
+give a _return value_. When writing code, sometimes we'll want our methods to
+evaluate code and _return_ a value -- not just output text, but return it to the
+the place that invoked the method.
 
-## SWBAT 2
+We'll cover the concept of return values in Ruby, and how this actually differs
+from `puts` and `print`commands.
+
+## Identify implicit return values in Ruby syntax
+
+By default, Ruby doesn't display any output, however, this doesn't mean that
+nothing is happening. Methods like `puts` and `print` allow us to output text to
+the console. This is actually different from Ruby's concept of a _return value_.
+
+A _return value_ is the data returned to the program by the execution of a
+method, the assignment of a variable, or other anything in Ruby. Everything has
```

Comment:
"or, generally, the evaluation of any _expression_ in Ruby. In Ruby, wherever there's an expression, there's a return value"

(we expl. expressions in great detail in new Prework bricks)